packages/Footer/Footer.vue

```diff
@@ -0,0 +1,170 @@
+<template>
+  <section id="footer">
+    <slot>
+      <div>
+        <div class="container">
+          <nav flex row space-between wrap mobile-column-reverse>
+            <div v-if="banner" flex column third>
+              <a v-if="banner.image" href="/">
+                <img :src="banner.textOrImgUrl" alt />
+              </a>
+              <div v-else>
+                <h2>{{banner.textOrImgUrl}}</h2>
+              </div>
+              <p class="copyright">{{copyRightText}}</p>
+              <a href="/" class="policy link-secondary" mb2>{{policyText}}</a>
+              <div v-if="!noIcons" flex space-between wrap>
+                <a v-for="(icon,i) in icons" :key="i" :href="icon.to" target="_blank" :class="icon">
+                  <i :class="icon.icon"></i>
+                </a>
+              </div>
+            </div>
+            <ul flex row mobile-column class="menu">
+              <li flex column quarter v-for="(link, i) in links" :key="i">
```

Comment:
`quarter` what if I have 5 columns or 3? could you make an example in the storybook for that